machine.go

```diff
@@ -112,13 +88,20 @@ type Config struct {
 	// to the microVM.
 	NetworkInterfaces []NetworkInterface
 
+	// CaptureFifoLogsToFile will copy the contents from the given fifo
```

Comment:
That makes sense. I am not too familiar with unlinking, but from the man pages it states that will delete a name from a file system if no process has it opened. Since we'll have it opened, unlinking would fail?

Reply:
No, the directory entry will be removed. `unlink()` doesn't know or care about open files. From the manual:

> If the name was the last link to a file but any processes still have the file open, the file will remain in existence until the last file descriptor referring to it is closed.

and

> If  the name referred to a socket, FIFO, or device, the name for it is removed but processes which have the object open may continue to use it. 

This is precisely the behavior we want.

Reply:
Done